src/colvarbias.cpp

```diff
@@ -93,11 +93,18 @@ int colvarbias::init(std::string const &conf)
     cvm::log("Reinitializing bias \""+name+"\".\n");
   }
 
+  feature_states[f_cvb_step_zero_data].available = true;
+
   colvar_values.resize(num_variables());
   for (i = 0; i < num_variables(); i++) {
     colvar_values[i].type(colvars[i]->value().type());
     colvar_forces[i].type(colvar_values[i].type());
     previous_colvar_forces[i].type(colvar_values[i].type());
+    if (!colvars[i]->is_enabled(f_cv_total_force_current_step)) {
+      // If any colvar does not have current-step total force, then
+      // we can't do step 0 data
+      feature_states[f_cvb_step_zero_data].available = false;
+    }
```

Comment:
`f_cvb_step_zero_data` is also a feature used in the absence of total forces, e.g. adding a hill at step 0 for metadynamics. I'm not a fan of enabling it for that use case, but I think this check would break it.

Reply:
Ok, I'm moving this check to `colvarbias_abf`.